packages/firebase_vertexai/firebase_vertexai/example/lib/main.dart

```diff
@@ -91,52 +91,77 @@ class _ChatWidgetState extends State<ChatWidget> {
 
     initFirebase().then((value) {
       _model = FirebaseVertexAI.instance.generativeModel(
-        model: 'gemini-1.5-flash-preview-0514',
+        model: 'gemini-1.5-flash',
       );
       _functionCallModel = FirebaseVertexAI.instance.generativeModel(
-        model: 'gemini-1.5-flash-preview-0514',
+        model: 'gemini-1.5-flash',
         tools: [
-          Tool(functionDeclarations: [exchangeRateTool]),
+          Tool.functionDeclarations([getWeatherTool]),
         ],
       );
       _chat = _model.startChat();
     });
   }
 
-  Future<Map<String, Object?>> findExchangeRate(
+  // This is a hypothetical API to return a fake weather data collection for certain location
+  Future<Map<String, Object?>> getWeather(
```

Comment:
I think it would be better to choose idiomatic Dart names than to match other languates. Can we replace the leading "get" with "fetch"? https://dart.dev/effective-dart/design#avoid-starting-a-method-name-with-get

Reply:
Updated